tests/filters/flogs.py

```diff
@@ -0,0 +1,311 @@
+import logging
+import os
+import uuid
+import ctypes
+import redis
```

Comment:
Maybe we can add a requirement.txt for test's python dependencies with redis and darwin python ?